lib/improver/tests/wind_downscaling/test_roughnesscorrection.py

```diff
@@ -37,16 +37,60 @@
 import iris
 from iris.coords import AuxCoord
 from iris.tests import IrisTest
+from iris.coord_systems import (LambertAzimuthalEqualArea,
```

Comment:
pylint returns an error for this: 
```
E: 40, 0: No name 'LambertAzimuthalEqualArea' in module 'iris.coord_systems' (no-name-in-module)
```
I've no idea why though

Reply:
It makes the same complaint in improver/lib/improver/tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py where it was initially defined. I have no idea why it complains either.